src/emulator/config/src/config.cpp

```diff
@@ -255,24 +256,36 @@ ExitCode init_config(Config &cfg, int argc, char **argv, const Root &root_paths)
             std::cout << window_title << std::endl;
             return QuitRequested;
         }
-        if (var_map.count("recompile-shader"))
+        if (var_map.count("recompile-shader")) {
             return QuitRequested;
-
-        if (cfg.load_config || cfg.config_path != root_paths.get_base_path()) {
-            if (cfg.config_path.empty()) {
-                cfg.config_path = root_paths.get_base_path();
+        }
```

Comment:
remove {} don't needed  here

Reply:
I originally added the braces for readability rather than for need (and because it looks cleaner when with the other Boost.Optional). This may change in a different commit, but if nothing comes out of it, I'll delete it.